tools/block_cache_trace_analyzer_test.cc

```diff
@@ -85,11 +91,12 @@ class BlockCacheTracerTest : public testing::Test {
     assert(writer);
     for (uint32_t i = 0; i < nblocks; i++) {
       uint32_t key_id = from_key_id + i;
+      uint32_t timestamp = (key_id + 1) * 1000000;
```

Comment:
Ditto. Maybe have a constant `kMicrosInSecond`.

Reply:
Thanks. Updated to use kMicrosInSecond.